bpython/repl.py

```diff
@@ -770,13 +773,13 @@ def line_is_empty(line):
             indentation = 0
         return indentation
 
-    def formatforfile(self, s):
+    def formatforfile(self, stdout):
```

Comment:
I agree `s` isn't great, but at least there's the convention that `s` is a string. Stdout sounds like a file descriptor or sys.stdout or something to me. How about `session_output`?